src/localization/index.ts

```diff
@@ -14,10 +14,13 @@ export const langNames = {
   'cs-cz': '🇨🇿',
   'fa': '🇮🇷',
   it: '🇮🇹',
+  'ar': '🇸🇦',
 };
 
 export const rtlSupported = ['fa'];
 
+export const rtlSupported = ['ar'];
```

Comment:
you should combine these array

```js
export const rtlSupported = ['fa', 'ar'];
```

Reply:
Done https://github.com/aykutkardas/regexlearn.com/pull/422/commits/d8254a0586c7b54b8c46e05a8bef0c279854fdae